addons/phone_validation/models/mail_thread_phone.py

```diff
@@ -56,11 +58,12 @@ def _search_phone_mobile_search(self, operator, value):
                   OR REGEXP_REPLACE(model.mobile, '[^\d+]+', '', 'g') SIMILAR TO CONCAT(%s, REGEXP_REPLACE(%s, '\D+', '', 'g'), '%%')
             """
 
-    # searching on +32485112233 should also finds 00485112233 (00 / + prefix are both valid)
-    # we therefore remove it from input value and search for both of them in db
+        # searching on +32485112233 should also finds 0032485112233
+        # we therefore remove it from input value and search for both of them in db
         if value.startswith('+') or value.startswith('00'):
-            value = value.replace('+', '').replace('00', '', 1)
-            starts_with = '00|\+'
+            if value.startswith('00'):
+                value = value[2:]
+            starts_with = r'(00|\+)'
```

Comment:
Remove these changes, as they are not part of the original specification and are not explained in your commit message.
Let's keep the diff minimal for this FIX and make it obvious we fix the mentioned issue.